homeassistant/components/mobile_app/webhook.py

```diff
@@ -137,9 +144,26 @@ def setup_registration(hass: HomeAssistantType, store: Store,
                                  headers=headers)
 
     if webhook_type == WEBHOOK_TYPE_UPDATE_LOCATION:
+        see_payload = {
+            ATTR_DEV_ID: registration[ATTR_DEVICE_ID],
+            ATTR_HOST_NAME: registration[ATTR_DEVICE_NAME],
```

Comment:
We shouldn't put in a host name, that's not the same as device name?

Reply:
```suggestion
```